<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1746</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>21</width>
      <height>661</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>1161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="Runningtable">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="Readytable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>571</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="Finishtable">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>561</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Runninglabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>程序运行队列</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>1161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableView" name="Suspendtable">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>561</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Suspendlabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>程序挂起队列</string>
    </property>
   </widget>
   <widget class="QTableView" name="Waitingtable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>571</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Waitinglabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>程序后备队列</string>
    </property>
   </widget>
   <widget class="QLabel" name="Readylabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>程序就绪队列</string>
    </property>
   </widget>
   <widget class="QLabel" name="Finishlabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>程序完成队列</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>571</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Choosepage">
     <widget class="QRadioButton" name="QRandomradioButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>随机生成作业</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="QInsertradioButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>115</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>手动插入作业</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="OKpushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="insertPRIlineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="insertPTIlineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请设置进程运行时间：</string>
      </property>
     </widget>
     <widget class="QSlider" name="PRIhorizontalSlider">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请设置优先权：</string>
      </property>
     </widget>
     <widget class="QSlider" name="PTIhorizontalSlider">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="InsertpushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>插入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IResetpushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IBackpushButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请设置进程运行资源：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="insertPSOlineEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="PSOhorizontalSlider">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>110</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="RunpushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>0</y>
      <width>20</width>
      <height>641</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableView" name="baseTableView">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>30</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="memoryTableView">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>340</y>
      <width>261</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>内存分配情况</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>Runningtable</zorder>
   <zorder>Readytable</zorder>
   <zorder>Finishtable</zorder>
   <zorder>Runninglabel</zorder>
   <zorder>line_3</zorder>
   <zorder>Suspendtable</zorder>
   <zorder>Suspendlabel</zorder>
   <zorder>Waitinglabel</zorder>
   <zorder>Readylabel</zorder>
   <zorder>Finishlabel</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>RunpushButton</zorder>
   <zorder>Waitingtable</zorder>
   <zorder>line_4</zorder>
   <zorder>baseTableView</zorder>
   <zorder>memoryTableView</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1746</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
